modules/packages/Zarr.chpl

```diff
@@ -174,7 +174,7 @@ module Zarr {
     const thisChunkDomain: domain(dimCount) = thisChunkRange;
     return thisChunkDomain;
   }
-  private proc getChunkDomain(chunkShape: ?dimCount*int, chunkIndex: int) {
+  proc getChunkDomain(chunkShape: ?dimCount*int, chunkIndex: int) {
```

Comment:
This should probably be documented or marked `nodoc` so it doesn't show up in the docs without a docstring